pkg/proxy/iptables/proxier_test.go

```diff
@@ -579,12 +671,22 @@ func assertIPTablesRulesEqual(t *testing.T, expected, result string) {
 	}
 
 	assert.Equal(t, expected, result)
+
+	err = checkIPTablesRuleJumps(expected)
+	if err != nil {
+		t.Fatalf("%s", err)
+	}
 }
 
 // assertIPTablesRulesNotEqual asserts that the generated rules in result DON'T match the
 // rules in expected, ignoring irrelevant ordering differences.
 func assertIPTablesRulesNotEqual(t *testing.T, expected, result string) {
-	expected, err := sortIPTablesRules(expected)
+	err := checkIPTablesRuleJumps(expected)
+	if err != nil {
+		t.Fatalf("%s", err)
+	}
```

Comment:
don't we have to check `checkIPTablesRuleJumps(result)` too ? 
we are asserting that expected and result are not equal so we can't infer from this that result rules are correct